src/main/resources/openshift/app/configuration/eap.cli.original

```diff
@@ -19,5 +19,9 @@ jms-topic add --topic-address=executorCancellation --entries=topics/executorCanc
 /subsystem=undertow/configuration=handler/file=windup-web-redirect:write-attribute(name=path,value=${jboss.home.dir}/windup-web-redirect)
 /subsystem=undertow/server=default-server/host=default-host/location=\//:write-attribute(name=handler,value=windup-web-redirect)
 
+/core-service=management/security-realm=HTTPSRealm/:add
```

Comment:
Do you know why this section is necessary? It seems like the S2I image should be adding the realm itself if the properties are set.